tests/copilot_helper/test_command_codebase.py

```diff
@@ -134,18 +143,19 @@ def test_codebase_prepare_generates_an_executable_image_build_run_file(self, tmp
 
 
 class TestCodebaseBuild:
-    @patch("boto3.client")
     @patch("click.confirm")
     @patch("subprocess.run")
-    @patch("dbt_copilot_helper.utils.application.get_aws_session_or_abort", return_value=boto3)
+    @patch("dbt_copilot_helper.commands.codebase.get_aws_session_or_abort")
```

Comment:
Its a little annoying that you've had to switch back to patching the command file rather than maintaining the patch on the dependency.

Reply:
This is because the function is imported as `from dbt_copilot_helper.utils.aws import get_aws_session_or_abort`. See: https://docs.python.org/3/library/unittest.mock.html#where-to-patch for details. I think it was probably working incidentally before because of the additional "@mock_ssm" annotation.